packages/pages/src/common/src/parsers/templateParser.test.ts

```diff
@@ -68,10 +68,83 @@ describe("makeClientTemplate", () => {
       `import { Template, TemplateRenderProps } from "@yext/pages";`
     );
   });
+
+  it("does not return tsdoc comments", () => {
+    const sourceParser = createParser(
+      `/**
+      * mock tsdoc comments
+      */
+      const foo = 5;
+      export default foo`
+    );
+    const testParser = createParser(``);
+    const templateParser = new TemplateParser(
+      sourceParser
+    ).makeClientTemplateFromSfp(testParser);
+    expect(templateParser.fileContents).not.toContain(` mock tsdoc comments`);
+  });
+});
+
+describe("complex makeClientTemplate usages", () => {
+  it("correctly returns expected file contents", () => {
+    const sourceParser = new SourceFileParser(
+      "./tests/fixtures/sourceFileTemplates/static.tsx",
+      createTsMorphProject()
+    );
+    const expectedParser = new SourceFileParser(
```

Comment:
You shouldn't need to use the parser against the client. You want to check that the regular template (static.tsx) returns exactly what is in your client file by just reading in the client and comparing it.